spec/shop_spec.rb

```diff
@@ -23,5 +23,12 @@
       #rspec matcher tests if object passed to expect is = to object passed to eq
     end
   end
-end
 
+  describe "#all" do
+    rows = DB[:shops].all
+    it 'should have same number of shops as db' do
```

Comment:
Now that we are no longer testing on the number of rows, we could change the name of this test to sth like:

`it 'should return all rows from the shops table`